build/cargo.gni

```diff
@@ -0,0 +1,35 @@
+# Runs a script to compile a local Cargo package and all of its dependencies
+template("cargo_build") {
+  action(target_name) {
+    script = "//brave/script/cargo.py"
+
+    # make sure rebuilds when files change
+    public_configs = invoker.public_configs
```

Comment:
we also need deps from the invoker so 
```
forward_variables_from(invoker,
                             [
                               "deps",
                               "inputs",
                               "public_deps",
                               "data_deps",
                             ])
```
or I think you can do `forward_variables_from(invoker, *)` and use things like manifest_path directly instead of calling invoker.manifest_path, but not 100% sure on that one

Reply:
Resolved